beacon-chain/sync/initial-sync/sync_state.go

```diff
@@ -47,17 +46,7 @@ func (s *InitialSync) processState(msg p2p.Message) {
 		return
 	}
 
-	exists, blkNum, err := s.powchain.BlockExists(ctx, bytesutil.ToBytes32(finalizedState.LatestEth1Data.BlockHash32))
```

Comment:
Why remove this? dont we still want to verify that the block exists?

Reply:
Good catch, adding it back.